packages/web3-providers-ws/src/index.ts

```diff
@@ -315,9 +316,7 @@ export default class Web3ProviderWS
                 }
 
                 if (id && this.responseQueue.has(id)) {
-                    let requestItem = this.responseQueue.get(id);
```

Comment:
Why is this not needed anymore?

Reply:
its already mentioned in title of [this](https://github.com/ChainSafe/web3.js/pull/4273/commits/e11b514358f4c84ac4b47864166e42ab6b7b0e9a) commit